### DISCUSSION TITLE
Releasing 9.0.0

### DISCUSSION THREAD

**Discussion by andresesfm** [hyochan/react-native-iap#1852] [General]
@hyochan, @jeremybarbet. I want to get your thoughts on releasing 9.0.0 as the last RC (4) was well received by people who tested it. 

I have prepared the following PR with all the changes needed for this: https://github.com/dooboolab/react-native-iap/pull/1843. Note that it is not a direct merge from next for the following reason @jeremybarbet brought up: https://github.com/dooboolab/react-native-iap/issues/1838 . I suggest the following:

1) Merge the new PR and release 9.0.0 to move forward
2) @hyochan to reset `next` to the new `main` branch 
3) @jeremybarbet to rebase and create PRs against the new `next` branch with his breaking changes in preparation for the next major release.

Let me know what you think

**Comment by jeremybarbet** [comment]
That's sounds good to me. Well done for publishing v9! 

It's also better to release, even major version, without thousands of changes. I will continue working on my 2 PRs I have open, and we can release them as v10 once completed.

2. I don't think that's necessary anymore, since v9 is now landed in main, we can just merge main into next, and use next for the next major release

**Reply by andresesfm** [reply]
@jeremybarbet 2 is still needed. There's no other way to reconcile main and next
